content/docs/command-reference/stage/list.md

```diff
@@ -10,55 +10,54 @@ usage: dvc stage list [-h] [-q | -v]
                       [targets ...]
 
 positional arguments:
-  targets          Show stages from a dvc.yaml file or a directory (with -R).
-                   'dvc.yaml' by default.
+  targets       Limit command scope to these dvc.yaml files,
+                directories (with -R), or stage names.
+                './dvc.yaml' by default.
 ```
 
+> See [`targets`](#options) for more details.
```

Comment:
we don't use anywhere else? let's not complicate it with Notes?

Reply:
We use it in https://dvc.org/doc/command-reference/repro @shcheklein. I remember we decided to put it there so the user doesn't miss the `targets` section in **Options**. Added here for that same reason and for consistency with the `repro` ref.

Reply:
kk